src/py_converters.cpp

```diff
@@ -94,6 +94,13 @@ int convert_from_attr(PyObject *obj, const char *name, converter func, void *p)
     return 1;
 }
 
+int convert_voidptr(PyObject *obj, void *p)
+{
+    void **val = (void **)p;
+    *val = PyLong_AsVoidPtr(obj);
+    return *val != NULL ? 1 : !PyErr_Occurred();
```

Comment:
nit: more readable as `val == NULL && PyErr_Occurred() ? 0 : 1`

Reply:
That's exactly why I wanted to leave it at the old version: I don't think this discussion really helps (and I think my version is more readable: if it's non-null, PyLong_AsVoidPtr succeeded and we don't need to check for an error, otherwise do check).
I could even write the whole thing as `return ((void **)p = PyLong_AsVoidPtr(obj)) ? 1 : !PyErr_Occurred();` but that's just silly.